packages/weather/OpenWeatherMap.py

```diff
@@ -43,32 +44,103 @@ def wrapper_load_config(string, entities):
         return func(payload)
     return wrapper_load_config
 
+
 def acquire_weather(func):
     @functools.wraps(func)
     def wrapper_acquire_weather(payload):
         for item in payload["entities"]:
-            if item["entity"] == "city":
+            if item["entity"] == "date" or item["entity"] == "datetime":
```

Comment:
What is this if case for? If one is asking for the present weather why would he need to specify a date? Unless you're trying to merge `acquire_weather` and `future`

Reply:
It's because Leon recognize future expressions like present expressions.
When you say "Tell me the weather in Paris tomorrow" he read that exactly if it was the weather right now.